csvcubed/csvcubed/utils/qb/validation/observations.py

```diff
@@ -147,11 +151,28 @@ def _validate_multi_measure_cube(
 ) -> List[ValidationError]:
     errors: List[ValidationError] = []
 
-    multi_measure_columns = get_columns_of_dsd_type(cube, QbMultiMeasureDimension)
+    multi_measure_columns: List[QbColumn[QbMultiMeasureDimension]] = get_columns_of_dsd_type(cube, QbMultiMeasureDimension)
     if len(multi_measure_columns) == 0:
         errors.append(NoMeasuresDefinedError())
     elif len(multi_measure_columns) > 1:
         errors.append(MoreThanOneMeasureColumnError(len(multi_measure_columns)))
+    else:
+        measure_column: QbColumn[QbMultiMeasureDimension] = multi_measure_columns[0]
+
+        all_measures_existing = all(
+            [
+                isinstance(m, ExistingQbMeasure)
+                for m in measure_column.structural_definition.measures
+            ]
+        )
+    
+        if all_measures_existing:
```

Comment:
it would be neater to just have one if block `if all_measures_existing and measure_column.csv_column_uri_template is None:`